http/vibe/http/server.d

```diff
@@ -1733,7 +1733,7 @@ private void listenHTTPPlain(HTTPServerSettings settings, HTTPServerContext cont
 
 
 private void handleHTTPConnection(TCPConnection connection, HTTPListenInfo listen_info)
-@safe {
+@trusted {
```

Comment:
I agree to that. Even more off-topic: `@trusted` functions are actually a D language misdesign. Would have been better to have unsafe blocks like C#. Fortunately, we can emulate those in D with `() @trusted { ... } ();`.
Okay, to get on topic: Further down this diff, such an `@trusted`-block was already there, and all changes are inside that block, so I don't really understand the reason of `@trusted` here.